GliaWidgets/ViewModel/Chat/ChatViewModel.swift

```diff
@@ -808,6 +807,6 @@ extension ChatViewModel {
 
     enum StartAction {
         case startEngagement
-        case none
+        case none(call: Call?)
```

Comment:
I'm sorry but I missed the sense/name for this case. StartAction is none with optional Call instance. For what purposes did you create this case? perhaps we could think about a more convenient name for this case?
Based on switch above this case is needed for call... but why did you name it `none`? (just try to understand your thinking).

Reply:
it was named before me, it basically means "don't do anything, since engagement is already ongoing"